taskcat/stacker.py

```diff
@@ -490,6 +490,27 @@ def get_available_azs(self, region, count):
             azs = ','.join(available_azs[:count])
             return azs
 
+    def get_single_az(self, region, az_id):
+        """
+        Get a single valid AZ for the region.
+
+        The number passed indicates the ordinal representing the AZ returned.
+        For instance, in the 'us-east-1' region, providing '1' as the ID would
+        return 'us-east-1a', providing '2' would return 'us-east-1b', etc.
+
+        In this way it's possible to get availability zones that are
+        guaranteed to be different without knowing their names.
+
+        :param region: Region of the availability zone
+        :param az_id: 0-based ordinal of the AZ to get
+
+        :return: The requested availability zone of the specified region.
+        """
+
+        regional_azs = self.get_available_azs(region, az_id)
```

Comment:
what's the benefit of the extra abstraction here over calling `ec2_client.describe_availibility_zones` directly?

Reply:
My goal was to add as little as possible. I'm still new to this codebase and wanted to make as small and easily understood an impact as possible.